src/jdk.attach/share/classes/sun/tools/attach/HotSpotVirtualMachine.java

```diff
@@ -399,20 +407,10 @@ void processCompletionStatus(IOException ioe, String cmd, InputStream sis) throw
                 throw new IOException("Protocol mismatch with target VM");
             }
 
-            // Special-case the "load" command so that the right exception is
```

Comment:
We had special logic for "load" command in 2 places (here and in the loadAgentLibrary method)
For simplification moved all logic to loadAgentLibrary()